Source/ISHRemote/Trisoft.ISHRemote/Connection/InfoShareWcfSoapWithWsTrustConnection.cs

```diff
@@ -350,17 +542,21 @@ public Annotation25ServiceReference.Annotation GetAnnotation25Channel()
 #if NET48
             if ((_annotationClient == null) || (_annotationClient.InnerChannel.State == System.ServiceModel.CommunicationState.Faulted))
             {
+                _logger.WriteDebug($"GetAnnotation25Channel recreating client - Client[{_annotationClient?.InnerChannel?.State}]");
```

Comment:
Rename WriteDebug `Client` to `ClientInnerChannelState`. This across all similar debug lines in this file.